trade-finance-manager-ui/server/controllers/helpers/validateSubmittedValues.helper.js

```diff
@@ -1,21 +1,15 @@
 const CONSTANTS = require('../../constants');
 
-const { hasValue, isAlphanumeric } = require('../../helpers/string');
+const { hasValue } = require('../../helpers/string');
 const { increment } = require('../../helpers/number');
 const generateValidationErrors = require('../../helpers/validation');
 
-const MAX_COMMENTS_LENGTH = 1000;
+const MAX_COMMENTS_LENGTH = 8000;
 
 const validateCommentField = (validationErrors, errorsCount, fieldLabel, fieldId, value) => {
   let errors = validationErrors;
   let count = errorsCount;
 
-  if (!isAlphanumeric(value)) {
-    count = increment(count);
-
-    errors = generateValidationErrors(fieldId, `${fieldLabel} must only include letters a to z, numbers, hyphens, commas and spaces`, count, errors);
-  }
-
   // remove new lines from textarea input value
   const strippedValueArray = value.split(/[\r]/g);
```

Comment:
Do you think I should remove this as well. 
It feels like something the user's would want, but I'm not sure why it was introduced originally. Perhaps we still need it for some reason?

Reply:
Happy for the above function (LOC:14) to be removed.